CodegenCLI/Tests/Commands/InitializeTests.swift

```diff
@@ -214,7 +215,7 @@ class InitializeTests: XCTestCase {
       let expectedPath = URL(fileURLWithPath: outputPath).standardizedFileURL.path
 
       expect(actualPath).to(equal(expectedPath))
-      expect(data).to(equal(self.expectedJSON.data(using: .utf8)!))
+      expect(data?.asString).to(equal(self.expectedJSON))
```

Comment:
This change is simply to get more readable test failure messages. Comparing strings will give us the human-readable string instead of this obscure failure message.
> failed - expected to equal <Data<hash=205197838,length=820>>, got <Data<hash=205197838,length=821>>